addons/website_slides/controllers/main.py

```diff
@@ -497,7 +497,7 @@ def slide_channel_tag_search_read(self, fields, domain):
     # --------------------------------------------------
 
     @http.route('''/slides/slide/<model("slide.slide", "[('website_id', 'in', (False, current_website_id))]"):slide>''', type='http', auth="public", website=True)
-    def slide_view(self, slide, **kwargs):
+    def slide_view(self, slide, search_category=None, search_slide_type=None, search_tag=None, search_uncategorized=None, **kwargs):
```

Comment:
Could all this stuff be fetched from the kwargs instead to avoid putting so many optional parameters in the method?
What do you think @jem-odoo ?